dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -357,6 +359,15 @@ func generateTarball(opts *GenerateTarballOpts) error {
 		run("adding Alluxio FUSE jar", "mv", fmt.Sprintf("integration/fuse/target/alluxio-integration-fuse-%v-jar-with-dependencies.jar", version), filepath.Join(dstPath, "integration", "fuse", fmt.Sprintf("alluxio-fuse-%v.jar", version)))
 		// Generate Helm templates in the dstPath
 		run("adding Helm chart", "cp", "-r", filepath.Join(srcPath, "integration/kubernetes/helm-chart"), filepath.Join(dstPath, "integration/kubernetes/helm-chart"))
+
+		// get ratis command line tools
+		if ratisShellSrc != "" {
```

Comment:
@Xenorith PTAL if this command line arg name makes sense to you. Do you prefer we generalize this to something like `--external-dep`? I'm fine with just calling this `ratis-shell-src`.

Reply:
if it is an externally sourced file, would it be more appropriate to append the bundled output tarball after it is generated via the script? ex. `tar -r alluxio.tgz ratis-shell`

if we allow this kind of change, then it would enable lots of other utilities to be bundled that are not alluxio specific. so i would prefer to avoid introducing the addition within the generation script when it is possible to be included afterwards